language/move-prover/boogie-backend/src/lib.rs

```diff
@@ -35,8 +35,8 @@ const VECTOR_SMT_ARRAY_EXT_THEORY: &[u8] =
     include_bytes!("prelude/vector-smt-array-ext-theory.bpl");
 const MULTISET_ARRAY_THEORY: &[u8] = include_bytes!("prelude/multiset-array-theory.bpl");
 
-const BCS_MODULE: &str = "0x1::BCS";
-const EVENT_MODULE: &str = "0x1::Event";
+const BCS_MODULE: &str = "Std::BCS";
```

Comment:
Undo this change should resolve the Boogie issue.

Reply:
Thanks for the spot! Definitely went a bit overboard with the find/replace